hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/StorageLocation.java

```diff
@@ -59,6 +61,8 @@
    *  e.g. [Disk]/storages/storage1/
    */
   private static final Pattern regex = Pattern.compile("^\\[(\\w*)\\](.+)$");
+  private static final Pattern CAPACITY_RATIO_REGEX =
```

Comment:
Similar with regex, let's add a java doc for CAPACITY_RATIO_REGEX. We can also consider rename "regex" to some more meaningful name.